test/libsolidity/semanticTests/arithmetics/exp_associativity.sol

```diff
@@ -0,0 +1,19 @@
+contract C {
+    // (2**3)**4 = 4096
+    // 2**(3**4) = 2417851639229258349412352
```

Comment:
Why not `return a**b**c == a**(b**c)`?

Reply:
I can add it. But it assumes that `a**(b**c)` is implemented correctly, though.

Reply:
You can also add another test that hard-codes the expected value in code. The problem with the current test I see is that people might accept the update without looking at the comment at the beginning of the file.